### DISCUSSION TITLE
Marketplace actions: additional insights and usage metrics

### DISCUSSION THREAD

**Discussion by jbergstroem** [community/community#38770] [Actions]
### Select Topic Area

Product Feedback

### Body

As a marketplace action author, it would be great to get deeper insights from who is using the action (how many times it is invoked, how long it takes to run, ..) as part of continously developing/maintaining it. There are obvious privacy boundaries here, but I think a few metrics that benefits both parties (developer, user) could be beneficial.

### Suggested solution

On the repository insights tab, add informaiton about times invoked, ci minutes spent, average runtime or similar. It could be solely based on public information inferred through repositories shown in "Used by" or anonymized from a bigger context.

**Comment by ianwilliams1** [comment]
Apparently there's private beta which may answer your questions, available under Org Insights...
![image](https://user-images.githubusercontent.com/45702008/222870029-7cb9f614-8f37-4961-b8a8-180558f3d5fa.png)

There are also additional charts for workflows and repos:

**Workflow**: workflow.yml
Satart-Date· End-Date
Action Activity
 -	Total Actions Minutes Usage Forecast
 -	Job Queue Time in Seconds
 -	Top Repositories by Daily Runtime Minutes
 -	Successful Runs by Repo Visibility
 -	Successful Runs by Runner Type
 -	Successful Runs by Runtime
 -	Top Invoking Events by Successful Workflow Run
	
	
**Repository**: repository-name
Satart-Date· End-Date
Action Activity
 -	Total Actions Minutes Usage Forecast
 -	Job Queue Time in Seconds
 -	Top Workflows by Daily Runtime Minutes
 -	Successful Runs by Runner Type
 -	Successful Runs by Runtime
 -	Top Invoking Events by Successful Workflow Run

Contact your Org's Customer Solutions rep.

**Reply by jsoref** [reply]
I don't suppose you have any idea how one could get a "Customer Solutions rep"? I'd really like one 🙂.

**Reply by ianwilliams1** [reply]
[Try Support - https://support.github.com/](https://support.github.com/contact), or 
[Enterprise Support - https://github.com/enterprise/contact](https://github.com/enterprise/contact)
